src/shared/atoms/MetaItem/MetaItem.tsx

```diff
@@ -0,0 +1,12 @@
+import React from "react";
+import "./MetaItem.css";
+import { type MetaItemProps } from "../../../types/index";
+
+export const MetaItem: React.FC<MetaItemProps> = ({ label, value }) => {
```

Comment:
Несколько рекомендаций: 
1. В современных приложухах принято импортировать типы из реакта отдельно и не использовать нотацию рода React.{какой-то тип}
2. FC в целом ок, но зачастую от него стараются уходить: он не несет никакой пользы, лишь добавляет дополнительный импорт. Зачастую сейчас пишут так: 
```suggestion
export const MetaItem = ({ label, value }: MetaItemProps) => {
```